app/build.gradle

```diff
@@ -2,10 +2,10 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    buildToolsVersion "25.4.0"
     defaultConfig {
         applicationId "com.codingblocks.oauthandroid"
-        minSdkVersion 15
+        minSdkVersion 18
```

Comment:
I increased `minSdkVersion` to implement UiAutomator as it requires `minSdkVersion` 18 or above